ts/components/ui/ListItemTransaction.tsx

```diff
@@ -0,0 +1,194 @@
+} from "./utils/baseListItems";
+
+type LogoNameOrUri = IOLogoPaymentType | ImageURISource;
+export type ListItemTransaction = WithTestID<
+  PressableBaseProps & {
+    hasChevronRight?: boolean;
+    isLoading?: boolean;
+    leftPaymentLogoOrUrl?: LogoNameOrUri;
+    subtitle: string;
+    title: string;
+  } & (
+      | {
+          transactionStatus: "success";
+          transactionAmount: string;
+        }
+      | {
+          transactionStatus: "failure" | "pending";
+          transactionAmount?: string;
+        }
+    )
+>;
+
+type LeftComponentProps = {
+  logoNameOrUrl: LogoNameOrUri;
+};
+
+const isImageUrI = (
+  value: IOLogoPaymentType | ImageURISource
+): value is ImageURISource =>
+  typeof value === "object" && value.uri !== undefined;
+
+const LeftComponent = ({ logoNameOrUrl }: LeftComponentProps) => {
+  if (isImageUrI(logoNameOrUrl)) {
+    return <Avatar shape="circle" size="small" logoUri={[logoNameOrUrl]} />;
+  } else {
+    return <LogoPayment size={44} name={logoNameOrUrl} />;
+  }
+};
+
+export const ListItemTransaction = ({
+  accessibilityLabel,
+  hasChevronRight = false,
+  isLoading = false,
+  leftPaymentLogoOrUrl,
+  onPress,
+  subtitle,
+  testID,
+  title,
+  transactionAmount,
+  transactionStatus = "success"
+}: ListItemTransaction) => {
+  const theme = useIOTheme();
+
+  if (isLoading) {
+    return <SkeletonComponent />;
+  }
+
+  const ListItemTransactionContent = () => {
+    const TransactionAmountOrBadgeComponent = () => {
+      switch (transactionStatus) {
+        case "success":
+          return (
+            <NewH6 color={hasChevronRight ? "blueIO-500" : "black"}>
+              {transactionAmount || "-"}
+            </NewH6>
+          );
+
+        case "failure":
+          return (
+            <IOBadge
+              color="red"
+              variant="solid"
+              labelColor="bluegreyDark"
+              text={I18n.t("global.badges.allCaps.failed")}
+            />
+          );
+        case "pending":
+          return (
+            <IOBadge
+              color="blue"
+              variant="solid"
+              labelColor="bluegreyDark"
+              text={I18n.t("global.badges.allCaps.onGoing")}
+            />
+          );
+      }
+    };
+
+    return (
+      <>
+        {leftPaymentLogoOrUrl && (
+          <View style={{ marginRight: IOListItemVisualParams.iconMargin }}>
+            <LeftComponent logoNameOrUrl={leftPaymentLogoOrUrl} />
+          </View>
+        )}
+        <View style={IOStyles.flex}>
+          <NewH6 color={theme["textBody-default"]}>{title}</NewH6>
+          <LabelSmall weight="Regular" color={theme["textBody-tertiary"]}>
+            {subtitle}
+          </LabelSmall>
+          <VSpacer size={4} />
+        </View>
+        <View style={Styles.rightSection}>
+          <TransactionAmountOrBadgeComponent />
+          {hasChevronRight && (
+            <Icon
+              name="chevronRightListItem"
+              color="blue"
```

Comment:
This icon has a `blue` color (part of the legacy palette), but the corresponding color of the `H6` above is `blueIO-500`. So we have two different blues at the same time. We should manage the different colors with the `isDesignSystemEnabled` boolean